Dockerfile

```diff
@@ -1,13 +1,6 @@
 FROM node:16-slim as static_files
 
 WORKDIR /code
-ENV PATH /code/node_modules/.bin:$PATH
-COPY package.json package-lock.json tailwind.config.js /code/
-RUN npm install --silent
```

Comment:
Ideally we can keep the node install and build in a separate Docker stage. What was the reason to move it into the deploy stage?

Reply:
Docker was broken locally for me until I did this. It’s been so long I forgot the root cause of why docker was broken locally

Reply:
I believe this breaks the `deploy` stage though:

```docker
 > [deploy 2/8] RUN npm install --silent:
#0 0.315 /bin/sh: 1: npm: not found
```

These changes got the build working for me again:

```diff
diff --git a/Dockerfile b/Dockerfile
index a55a5c2..e0f6888 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -1,5 +1,14 @@
 FROM node:16-slim as static_files
 
+WORKDIR /code
+ENV PATH /code/node_modules/.bin:$PATH
+COPY package.json package-lock.json tailwind.config.js /code/
+RUN npm install --silent
+COPY ./public /code/public/
+COPY ./src /code/src/
+WORKDIR /code/
+RUN npm run build
+
 WORKDIR /code
 
 FROM python:3.8-slim-bullseye as base
@@ -85,13 +94,8 @@ RUN touch /code/.env
 
 FROM base AS deploy
 
-ENV PATH /code/node_modules/.bin:$PATH
-COPY package.json package-lock.json tailwind.config.js /code/
-RUN npm install --silent
-COPY ./public /code/public/
-COPY ./src /code/src/
-WORKDIR /code/
-RUN npm run build
+# Copy in node-built files
+COPY --from=static_files /code/build /code/build
 
 # Create a group and user to run our app
 ARG APP_USER=appuser
diff --git a/src/components/elements/Input/FormTextArea.jsx b/src/components/elements/Input/FormTextArea.jsx
index c7f72fe..45ef1a4 100644
--- a/src/components/elements/Input/FormTextArea.jsx
+++ b/src/components/elements/Input/FormTextArea.jsx
@@ -1,4 +1,3 @@
-/* eslint-disable jsx-a11y/label-has-associated-control */
 import React from 'react';
 import cx from 'classnames';
 import { useController } from 'react-hook-form';

```

Reply:
Yeah I forgot that I modified the Dockerfile. I think I got annoyed while trying to debug the `label-has-associated-control` issue.

The problem with your change is that it breaks docker locally for me as the lint rule `jsx-a11y/label-has-associated-control` would fail.

Additionally, I think it's very odd that while building within docker it complains that the rule doesn't exist. Docker should be pulling the same dependencies and versions that we would use locally right? It seems to be using an out of date version of the eslint jsx-a11y rules

Reply:
All of this is to say that I want to fix whatever is happening before merging this PR in

Reply:
Fixed by removing the commented out eslint rule. Here's what I wrote on slack:

> I think it's because we don't copy in the .eslntrc.json file in the deploy step. but i don't really want the linting to be done while building the dockerfile anyway. something to work on later